### PR TITLE
BCME and Playful Ignitions

### PR DESCRIPTION
**Pull request by pianoblook**
Not sure why all the Dark Pink stuff is also showing up as 'changes', since those have already been added.

### CODE REVIEW

**Comment by Zamiell** [comment]
looks like you need to rebase your branch on master

**Comment by Zamiell** [comment]
do this
https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository

**Comment by pianoblook** [comment]
I feel like I understand about half of the words being said in that discussion thread. 

I'll see what I can do @_@

**Comment by Zamiell** [comment]
call me on discord

File: docs/extras/ejections.md [hunk 1]
```diff
@@ -30,35 +30,16 @@ import SelfColor from '@site/image-generator/yml/extras/ejections/self-color.yml
...
+### The Bad Chop Move Ejection
 
-- Normally, when known trash is touched as the focus of the clue, and the known trash is not on chop, it communicates a *Trash Chop Move*.
-- However, what if the *Chop Moved* cards are also all trash? The other players can see that this clue must have some other purpose.
-- In this situation, it should signal an *Ejection* on the very next player.
+- Many moves result in one or more cards becoming *Chop Moved*. However, what if such a move is used when the *Chop Moved* card(s) are all trash? The other players can see that this move must have been done for some other purpose.
+- In this situation, it should signal an *Ejection* on the very next player. 
 - This results in a *Signal Shift* from *Save* --> *Trash*.
-- *Bad Chop Move Ejection* can be performed throughout the game. (But keep in mind that in the *End-Game*, a player might just be stalling.)
+- **Any** move that results in a *Chop Move* can be used to initiate such an *Ejection*. These include, but may not be limited to, *Trash Chop Moves*, *5's Chop Moves*, *Tempo Clue Chop Moves*, *Chop Transfers*, and *Skipped Order Chop Moves*.
```
Comment (Zamiell) [inline docs/extras/ejections.md:38]: dont enumerate, as this can get out of date

File: docs/extras/pushes-pulls.md [hunk 2]
```diff
@@ -30,26 +30,46 @@ import FiveDoublePull from '@site/image-generator/yml/extras/five-double-pull.ym
 
 <br />
 
-### The 5 Double Pull (Or The Double 5 Pull)
+### The Playful Ignition
```
Comment (Zamiell) [inline docs/extras/pushes-pulls.md:33]: this move does not belong in a file called pushes-pulls, move it to rarely used

File: docs/extras/pushes-pulls.md [hunk 3]
```diff
@@ -30,26 +30,46 @@ import FiveDoublePull from '@site/image-generator/yml/extras/five-double-pull.ym
...
   - Save two important cards when there are a lot of cards to deal with and the team is low on clues.
   - Get a blind-play from *Elimination* when the duplicate is visible.
-- More examples of a *5 Double Pull* can be found [here](../examples/5-double-pull.md).
+- More examples of a *Playful Ignition* can be found [here](../examples/5-double-pull.md).
+
+<br />
+
+#### Interaction with 5 Rank Clues
+
+- This interaction is turned **off** in *Late-Game*
```
Comment (Zamiell) [inline docs/extras/pushes-pulls.md:60]: late-game does not exist

File: docs/extras/pushes-pulls.md [hunk 4]
```diff
@@ -30,26 +30,46 @@ import FiveDoublePull from '@site/image-generator/yml/extras/five-double-pull.ym
...
   - Avoid violating *Good Touch Principle*.
   - Save a *Delayed Playable* card that is followed by a critical card.
   - Save two important cards when there are a lot of cards to deal with and the team is low on clues.
   - Get a blind-play from *Elimination* when the duplicate is visible.
-- More examples of a *5 Double Pull* can be found [here](../examples/5-double-pull.md).
+- More examples of a *Playful Ignition* can be found [here](../examples/5-double-pull.md).
+
+<br />
+
+#### Interaction with 5 Rank Clues
```
Comment (Zamiell) [inline docs/extras/pushes-pulls.md:58]: the interaction should be referenced from the 5cm section

**Comment by Zamiell** [comment]
furthermore, your build fails the linter, make it pass

**Comment by pianoblook** [comment]
Strangely, though, the interaction only comes into existence once you're using Playful Ignitions *and* BCMEs, so I think we'd need a new 5cm section for max level

**Comment by Zamiell** [comment]
>  so I think we'd need a new 5cm section for max level

negative, make a new bullet point similar to what is done in this section https://hanabi.github.io/docs/level-1#the-5-stall

**Comment by pianoblook** [comment]
gotcha, good idea. I'll fix everything up this afternoon

**Comment by Zamiell** [comment]
instead of putting it in Misc page, lets put it in a new page called Ignitions, i think that makes more sense

**Comment by Zamiell** [comment]
ping pianoblook

**Comment by pianoblook** [comment]
oh, sure. I can do that ASAP (this week)

**Comment by Zamiell** [comment]
can you fix the broken links that the linter reports

File: docs/level-20.md [hunk 5]
```diff
@@ -32,10 +32,6 @@ title: Level 20 - Ignition
 
 <br />
 
-## Special Moves
```
Comment (Zamiell) [inline docs/level-20.md:35]: don't remove this

File: docs/level-20.md [hunk 6]
```diff
@@ -22,7 +22,7 @@ title: Level 20 - Ignition
 
 <br />
 
-### Double Ignition
```
Comment (Zamiell) [inline docs/level-20.md:25]: why did you change this

**Comment by Zamiell** [comment]
pinging piano

**Comment by pianoblook** [comment]
oh wow, totally missed this - sorry!

I'll try and fix this stuff in the next day or two

File: docs/level-20.md (hunk 6 above)
Comment (pianoblook) [reply docs/level-20.md:25]: okay I see - I changed this because it seems to be a parent section to both Double Ignition + Playful Ignition - both are types of 'double ignitions'. I guess we could also have the main category be 'Ignitions' and then separate them all out

File: docs/level-20.md (hunk 5 above)
Comment (pianoblook) [reply docs/level-20.md:35]: I don't understand what value this has - each of the 'Special Moves' listed are just different types of Double Ignitions - seems redundant, no?

Comment (Zamiell) [reply docs/level-20.md:35]: every section follows this format, so you should keep it for consistency
